vector-im/element-android
# [Session manager] Multi selection in sessions list (PSG-852)

## Type of change


## Content


Adding multi selection capability in session list screen. For now, we can do nothing with the selected sessions.

## Motivation and context


Closes #7396 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/196740168-b6d9c2f0-e9bb-4dd6-9442-ad59119547f3.gif" width=300 />

## Tests



- Enable the labs flag for the new Session manager
- Go to Settings -> Security & Privacy -> Show all sessions
- Press the "View all" button in the other sessions section (need at least 6 other sessions)
- Open the overflow menu of the screen
- Press "Select session"
- Check you can select sessions in the list and that toolbar is correctly updated
- Press back
- Check the select mode is disabled
- Long press a session item
- Check the select mode is enabled and selection is empty
- Check you can select sessions in the list and that toolbar is correctly updated
- Open the overflow menu of the screen
- Press the "Select all" option
- Check all sessions in the list are selected
- Open the overflow menu of the screen
- Press the "Deselect all" option
- Check no session in the list are selected
- Press back
- Check the select mode is disabled
- Press back
- Check the main screen of Session manager is displayed

## Tested devices

- OS version(s): Android 11

## Checklist
